jpegenc/subblocks/quantizer/quantiser.py

```diff
@@ -0,0 +1,154 @@
+"""The above module is the hardware implementation
+    of quantizer top module"""
+
+from myhdl import always_seq, block, always_comb
+from myhdl import Signal, concat, modbv, intbv
+
+from .ramz import ramz
+from .quantiser_core import (quantizer, QuantOutputStream,
+                             QuantInputStream, QuantConfig)
+
+
+class QuantIDataStream(object):
```

Comment:
I am confused, this is not a streaming interface?  This is an interface to a random-access-memory (RAM).  You have been indicating that this takes a streaming interface?  Are you intending on adding a buffer in front of this ~~module~~ block?